cmd/ci-operator/main.go

```diff
@@ -590,7 +590,16 @@ func (o *options) initializeNamespace() error {
 		for key, value := range updates {
 			update.ObjectMeta.Annotations[key] = value
 		}
-		if _, err := client.Namespaces().Update(update); err != nil {
+		var updateErr error
+		for retries := 5; retries > 0; retries-- {
+			if _, err := client.Namespaces().Update(update); err == nil {
```

Comment:
this won't work, you have to actually re fetch and reapply your changes.

Reply:
Oh ugh, we should just use patch. Why is that hard today with client libraries?